### PR TITLE
add cloud rebrand properties

### PR DESCRIPTION
**Pull request by gcusnieux**
## :id: Reference related issue. 

https://gravitee.atlassian.net/browse/CJ-1221

## :computer: Add screenshots for UI

Screenshot 2024-05-28 at 09 29 41
Screenshot 2024-05-28 at 09 29 37

## :man: :woman:@mentions of the person or team responsible for reviewing proposed changes

This is a nice example: https://github.com/gravitee-io/gravitee-access-management/pull/1822

### CODE REVIEW

File: gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/filter/CockpitAuthenticationFilter.java [hunk 1]
```diff
@@ -154,41 +154,45 @@ public void doFilter(jakarta.servlet.ServletRequest request, jakarta.servlet.Ser
     }
 
     private boolean enabled() {
-        return configuration.getProperty("cockpit.enabled", Boolean.class, false);
+        return getProperty("cockpit.enabled", "cloud.enabled", Boolean.class, false);
```
Comment (leleueri) [inline gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/filter/CockpitAuthenticationFilter.java:157]: **question**: do we have to test the property starting with cockpit first ? :thinking:  I was thinking the cloud one should be tested first and then fallback to the cockpit one at it is legacy. (this comment is for all the changes related to this getProperty method)

Comment (gcusnieux) [reply gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/filter/CockpitAuthenticationFilter.java:157]: Hi @leleueri 
We don't wish to introduce any breaking change which is why cockpit configuration takes precedence over cloud conf.

Comment (leleueri) [reply gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/filter/CockpitAuthenticationFilter.java:157]: ok, as there is a fallback IMO this is not a BC but lets keep it like this.

**Review by leleueri — APPROVED** [review]

File: gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/filter/CockpitAuthenticationFilter.java (hunk 1 above)
Comment (gcusnieux) [reply gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/filter/CockpitAuthenticationFilter.java:157]: Not really, depending of the installation type. 
If a user have `cockpit.enabled = true` and process to upgrade, the user will have both `cockpit.enabled = true` and `cloud.enbaled = false`

**Pull request merged**